package.json

```diff
@@ -10,7 +10,8 @@
     "build": "quasar build",
     "lint": "eslint --ext .js,.ts,.vue ./",
     "lint:fix": "eslint --fix --ext .js,.vue src",
-    "test": "jest --coverage"
+    "test": "jest --coverage",
+    "go": "jest test/jest/__tests__/antelope/stores/balances.spec.ts"
```

Comment:
do we need `go`?

Reply:
See my comment: https://github.com/telosnetwork/telos-wallet/pull/465#discussion_r1260038070

Reply:
Thanks both. I'm removing the script because is not needed anymore.